Firestore/core/test/firebase/firestore/remote/serializer_test.cc

```diff
@@ -235,8 +237,8 @@ class SerializerTest : public ::testing::Test {
     reader.ReadNanopbMessage(
         google_firestore_v1_BatchGetDocumentsResponse_fields, &nanopb_proto);
     serializer.DecodeMaybeDocument(&reader, nanopb_proto);
-    reader.FreeNanopbMessage(
-        google_firestore_v1_BatchGetDocumentsResponse_fields, &nanopb_proto);
+    FreeNanopbMessage(google_firestore_v1_BatchGetDocumentsResponse_fields,
```

Comment:
Not for this PR, but consider making the constructor for `Message` public so that we can write these tests in terms of it.